src/MessagingBubble/MessagingBubble.less

```diff
@@ -1,29 +1,39 @@
 @import (reference) "../less/index";
 
-.MessagingBubble--Message--Own {
-  background-color: @neutral_dark_gray;
+// shared styling across bubble variations
+.MessagingBubble--Message {
   border-radius: 0.3125rem;
-  color: @neutral_white;
   font-size: 1rem;
   max-width: 25rem;
-  overflow-wrap: break-word;
   .padding--xs();
   .text--line-height-4();
+  overflow-wrap: break-word;
   white-space: pre-wrap;
   word-wrap: break-word; // IE name for overflow-wrap
 }
```

Comment:
nit: add a newline here?

Reply:
tried adding a new line and the linter is not a huge fan